asciidoc/courses/aura-dashboards/modules/4-sharing-dashboards/lessons/3-further-steps/lesson.adoc

```diff
@@ -2,37 +2,178 @@
 :type: lesson
 :order: 3
 
-In the previous lessons, you learned how to manage your Aura instances and users.
+In the previous lessons, you learned how to create, customize, and share dashboards.
 
-In this section, you will learn about additional resources and next steps to continue your journey with Neo4j Aura.
+In this section, you will learn about additional resources and next steps to continue your journey with Neo4j Dashboards.
 
 Feel free to ask any questions in the chat!
 
 
+== Final Project: Build Your Stakeholders Dashboard
+
+Now it's time to apply everything you've learned by creating a comprehensive, production-ready dashboard.
+
+
+=== Project Overview
+
+**Scenario:** Design a dashboard for a streaming service to support content acquisition decisions. The dashboard will serve three stakeholder groups: executives, marketing team, and content acquisition team.
```

Comment:
I would call the lesson "Final Project..." and put a `read` button at the bottom so it triggers the completion modal.  Otherwise it seems a bit disjointed